build/docker/Datadog.Trace.ClrProfiler.IntegrationTests.sh

```diff
@@ -34,4 +34,6 @@ wait-for-it elasticsearch5:9200 -- \
 wait-for-it sqlserver:1433 -- \
 wait-for-it mongo:27017 -- \
 wait-for-it postgres:5432 -- \
+wait-for-it -t 60 kafka-zookeeper:2181 -- \
+wait-for-it -t 60 kafka-broker:9092 -- \
```

Comment:
I like that you added a timeout, we should probably consider adding a more reasonable one for each